globals.php

```diff
@@ -56,6 +56,10 @@
 $db->connect();
 $c = $db->connection_id;
 $set = get_site_settings();
+if ($set['use_timestamps_over_crons']) {
+    define('SILENT_CRONS', true);
+    require_once __DIR__ . '/crons/cronless_crons.php';
```

Comment:
Even though the eventual `run` call is error handled, I think there'd be value of try catching this whole require as to not blow up a users page if something else were to fail

Reply:
Addressed by [969d72383f9e0366b3e6a226e9ef0450831b21e0](https://github.com/Orsokuma/mccodesv2-php8/commit/969d72383f9e0366b3e6a226e9ef0450831b21e0)